### PR TITLE
generate a session secret

### PR DESCRIPTION
**Pull request by homeyer**

### CODE REVIEW

File: bin/install.sh [hunk 1]
```diff
@@ -339,6 +346,7 @@ echo -e "\n\nGreat! That's all we need, we will have your Waffle.io Takeout read
 # Store off environment configuration in /etc/waffle #
 ######################################################
 echo -e "\n\nSaving environment configuration in /etc/waffle/"
+sudo mkdir -p /etc/waffle
```
Comment (burtonjc) [inline bin/install.sh]: I don't think you need `-p`, `/etc` will always exist.

File: bin/install.sh [hunk 2]
```diff
@@ -316,6 +316,13 @@ then
   echo WAFFLE_DB_ENCRYPTION_KEY=$encryptionkey >> $envFile
 fi
 
+# WAFFLE_SESSION_SECRET
+if [ -z "$WAFFLE_SESSION_SECRET" ];
+then
+  sessionSecret=$(openssl rand -base64 32)
+  echo WAFFLE_SESSION_SECRET=$sessionSecret >> $envFile
+fi
+
```
Comment (burtonjc) [inline bin/install.sh]: Ugh, we need a docker build to be a part of CI so we catch this stuff sooner.

File: bin/install.sh (hunk 1 above)
Comment (homeyer) [inline bin/install.sh]: `mkdir` throws errors if it already exists, the `-p` makes it not display an error.

File: bin/install.sh (hunk 2 above)
Comment (homeyer) [inline bin/install.sh]: yep!

File: bin/install.sh (hunk 1 above)
Comment (burtonjc) [inline bin/install.sh]: I see

**Comment by burtonjc** [comment]
:+1:

**Comment by burtonjc** [comment]
Thanks for doing this.

**Pull request merged**